pkg/network/payload/notary_request.go

```diff
@@ -114,6 +115,29 @@ func (r *P2PNotaryRequest) isValid() error {
 	if nKeysMain[0].Value.(*transaction.NotaryAssisted).NKeys == 0 {
 		return errors.New("main transaction should have NKeys > 0")
 	}
+	if len(r.MainTransaction.Signers) < 2 {
```

Comment:
Why do we need this in payload validation? It should only care about fallback transaction to be correct, even if the main one is total garbage the fallback will still get in and that's the only thing regular node should care about. It's important to do as little things as safely possible on regular node, notaries should do their job in handling all of this, they're paid for it.

Reply:
I dropped this commit and some of these checks. The rest of them are moved to the notary module.